src/components/tables/DataList.js

```diff
@@ -60,7 +60,11 @@ export default class DataList extends React.PureComponent {
   }
 
   render() {
-    const { columnRenderers, columns, dataSource, onRow } = this.props;
+    const { columnRenderers, columns, dataSource, onRowClick } = this.props;
+
+    const onRow = (record, index) => ({
```

Comment:
Is there an `index`? According to https://ant.design/changelog there's only `record`

Reply:
Yep, there is an index. They didn't do a great job documenting this.